api/osrd_infra/models/train_schedule.py

```diff
@@ -17,3 +17,4 @@ class TrainScheduleModel(models.Model):
     mrsp = models.JSONField(validators=[JSONSchemaValidator(limit_value=MRPS.schema())])
     base_simulation = models.JSONField()
     eco_simulation = models.JSONField(null=True)
+    tags = models.JSONField(null=True)
```

Comment:
Why not using an ArrayField like the rolling stock does?

If we keep a JSONField:
- We need a schema. 
- The field can have a default value to `[]` instead of allowing `null`.